### PR TITLE
docs: fix link to v7 docs

### PR DESCRIPTION
**Pull request by shinGangan**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

The link to the v7 document in README.md was outdated and has been corrected.

Renovate config has also been modified.
  - `config:base` is renamed to `config:recommended`.

ref: https://github.com/renovatebot/renovate/pull/21136

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by shinGangan** [comment]
Hi @kazupon, please review when you have a time ☕️
This PR is low priority due to modification of README.md.

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The i18n module for Nuxt 3
 
 ## ✅ Status: v8
 
-If you would like to use the stable version for Nuxt 2, please see the [`v7` branch](https://github.com/nuxt-modules/i18n/tree/main)
+If you would like to use the stable version for Nuxt 2, please see the [`v7` branch](https://github.com/nuxt-modules/i18n/tree/v7)
```
Comment (kazupon) [inline README.md:17]: good catch!
thanks!

File: renovate.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base", ":prHourlyLimit4", ":semanticCommitTypeAll(chore)"],
+  "extends": ["config:recommended", ":prHourlyLimit4", ":semanticCommitTypeAll(chore)"],
```
Comment (kazupon) [inline renovate.json:3]: Oh, I did not know about it 😅
Thanks to you I know one more thing about renovate 💚

**Review by kazupon — APPROVED** [review]
LGTM!

**Pull request merged**